lib/local_authorities_url_importer.rb

```diff
@@ -50,23 +49,4 @@ def parse_url(url)
       "http://" + url
     end
   end
-
```

Comment:
Could you rebase these removals back into the URL importer commit, so that they disappear from history?